localstack/services/dynamodb/provider.py

```diff
@@ -1136,7 +1147,12 @@ def table_exists(account_id: str, region_name: str, table_name: str):
             aws_secret_access_key=TEST_AWS_SECRET_ACCESS_KEY,
             region_name=region_name,
         )
-        return aws_stack.dynamodb_table_exists(table_name, client)
+        return dynamodb_table_exists(table_name, client)
+        # try:
+        #     client.describe_table(TableName=table_name)
```

Comment:
Q: why was this code added and commented out?

Reply:
Weirdly  enough if line 1150 is replaced with the commented out code, all CI runs will hang and timeout :D 

I want to investigate why later in a follow-up so I haven't removed it yet